mappings/net/minecraft/test/GameTest.mapping

```diff
@@ -31,3 +42,9 @@ CLASS net/minecraft/class_4517 net/minecraft/test/GameTest
 	METHOD method_23638 getStructureName ()Ljava/lang/String;
 	METHOD method_23639 start ()V
 	METHOD method_23640 complete ()V
+	METHOD method_29402 getRotation ()Lnet/minecraft/class_2470;
+	METHOD method_29403 getTestFunction ()Lnet/minecraft/class_4529;
+	METHOD method_32240 clearArea ()V
+	METHOD method_32241 isFlaky ()Z
```

Comment:
Is this "flaky" in the literal strings? If not more formal words like "confused" "erroneous" would be better

Reply:
![image](https://user-images.githubusercontent.com/16228338/106352454-8e558680-62e3-11eb-9656-0519a072c3a7.png)

Reply:
As Yanis posted, Flaky is the official term for a test that may need multiple attempts.